addons/web/static/src/js/fields/relational_fields.js

```diff
@@ -767,6 +767,32 @@ var FieldMany2One = AbstractField.extend({
 
 var ListFieldMany2One = FieldMany2One.extend({
 
+    /**
+     * Should never be allowed to be opened while in readonly mode in a list
+     *
+     * @override
+     */
+    init: function () {
+        this._super.apply(this, arguments);
+        this.nodeOptions.no_open = this.mode === 'readonly' || this.nodeOptions.no_open;
```

Comment:
this is either useless, or it doesn't work, because we never access this.nodeOptions.no_open after init (we use this.noOpen instead)